src/reaper_osara.cpp

```diff
@@ -3583,6 +3583,26 @@ void cmdUnselAllTracksItemsPoints(Command* command) {
 		outputMessage(translate("unselected tracks/items/envelope points"));
 }
 
+void reportTempoTimeSig() {
+ReaProject* proj=EnumProjects(-1, nullptr, 0);
+if(proj==nullptr) return;
+double tempo=0;
+int timesig1=0;
+int timesig2=0;
+double pos=GetPlayPosition();
+TimeMap_GetTimeSigAtTime(proj, pos, &timesig1, &timesig2, &tempo);
+outputMessage(formatDouble(tempo, 1, false)+", "+to_string(timesig1)+"/"+to_string(timesig2));
+}
+
+void cmdManageTempoTimeSigMarkers(Command* command) {
+if(lastCommandRepeatCount==0)
+{
+reportTempoTimeSig();
+return;
+}
+Main_OnCommand(40256, 0);
```

Comment:
It'd be great if you could add a comment at the end of this line to explain what action this is:

```
Main_OnCommand(40256, 0); // Tempo envelope: Insert tempo/time signature change marker at edit cursor...
```